lib/beaker/dsl/install_utils/foss_utils.rb

```diff
@@ -367,6 +367,12 @@ def install_puppet_agent_on(hosts, opts = {})
             end
 
             if package_name
+              # Some SLES images already have rubygem-puppet installed and this
```

Comment:
@tvpartytonight What I'm getting at is that the Beaker code itself calls that before I even get to my pre-suite anything from what I remember and that's a bit beyond the level of Beaker mangling that I'd like to do.